sdk/iot/provisioning/samples/src/paho_iot_provisioning_example.c

```diff
@@ -262,6 +220,119 @@ static int register_device()
   return 0;
 }
 
+static void print_buffer(az_span span)
+{
+  char* buffer = (char*)az_span_ptr(span);
+  for (int i = az_span_size(span); i > 0; i--)
```

Comment:
Why loop backwards?

Reply:
I know I had a good reason in some iteration :D.